### PR TITLE
Work on supporting python 3

### PR DESCRIPTION
**Pull request by streeter**

### CODE REVIEW

File: webapp2.py [hunk 1]
```diff
@@ -119,7 +123,7 @@
     505: 'HTTP Version not supported',
 }
 status_reasons.update(_webapp_status_reasons)
-for code, message in _webapp_status_reasons.iteritems():
+for code, message in _webapp_status_reasons.items():
```
Comment (jonparrott) [inline webapp2.py]: Any performance concerns here?

**Comment by jonparrott** [comment]
Thanks for this!

File: webapp2.py (hunk 1 above)
Comment (streeter) [inline webapp2.py]: `.items()` isn't an iterator in Python 2, so if `_webapp_status_reasons` is super huge, then we load them all into memory. However, it looks like the list isn't that large (7 items).

Comment (jonparrott) [inline webapp2.py]: How about these other instances? We need to be sure (or write a helper like `six.iteritems`)

Comment (streeter) [inline webapp2.py]: Ah, good idea. I'm happy to write that and add to the PR

Comment (streeter) [inline webapp2.py]: Fixed in 43547f29e69fbc9ce9ec4de755ade11b559e4eff

File: webapp2.py [hunk 2]
```diff
@@ -72,6 +72,13 @@
                     "so webapp2 won't be thread-safe!")
 
 
+def iteritems(d):
```
Comment (jonparrott) [inline webapp2.py]: unless this is intended to become part of the public API of webapp2, please name it `_iteritems`.

Comment (streeter) [inline webapp2.py]: Thanks! Fixed in 50e93a9

**Comment by jonparrott** [comment]
Great, this looks good to me. Thank you for contributing.

**Pull request merged**